src/session.cc

```diff
@@ -244,10 +439,31 @@ Session::GetAttributeValue(CK_OBJECT_HANDLE hObject,
 
     case CKA_LABEL:
       config_.debug_log("   Attribute %d: Label (unsupported)", i);
-      if (pTemplate[i].pValue) {
-        *static_cast<char*>(pTemplate[i].pValue) = 'x';
+      if (hObject == 1) {
```

Comment:
Does 1 and 2 have symbolic values?

Reply:
These are just my educated guess from the implementation of `object_class` and `FindObjects`. It seems like 1 represent public key and 2 represent private key. It is not documented anywhere. Feel free to document it or make it a bit more clear in the code.